.kokoro/appengine-datastore-flex.cfg

```diff
@@ -0,0 +1,16 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Download secrets from Cloud Storage.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/nodejs-docs-samples"
```

Comment:
This could be moved to a `common.cfg` file, then it wouldn't need to be copied into every `.cfg` file.